pkgs/development/ocaml-modules/dune-configurator/default.nix

```diff
@@ -1,17 +1,22 @@
-{ lib, buildDunePackage, dune_2, csexp, result }:
+{ lib, buildDunePackage, dune_3, csexp }:
 
 buildDunePackage rec {
   pname = "dune-configurator";
 
-  inherit (dune_2) src version patches;
+  inherit (dune_3) src version patches;
 
-  duneVersion = "2";
+  # This fixes finding csexp
+  postPatch = ''
+    rm -rf ./vendor
```

Comment:
With Dune, the particular vendored dependencies should be carefully checked in case they are being pointed at forks. In this case, csexp is itself maintained by the Dune team so it won't be a problem. However in the next version (3.8) I know that this particular dependency got bumped so we should make sure to bump csexp too.

Reply:
This is done for multiple dune packages and I was even told by @anmonteiro that this is correct.

Reply:
no, I said `rm -rf vendor/pp vendor/csexp`

Reply:
BTW this is not what I said or not, it's the [actual dune build command](https://github.com/ocaml/opam-repository/blob/master/packages/dune-build-info/dune-build-info.3.7.0/opam#L23-L38)

Reply:
Sorry, I misunderstood, I should change to follow that. Thanks for pointing it out!